src/utils.ts

```diff
@@ -284,13 +285,15 @@ const queryStringParser = (
         urlParams = queryStringParserFallback(url);
     }
 
+    urlParams.forEach((value, key) => {
+        lowerCaseUrlParams[key.toLowerCase()] = value;
+    });
+
     if (isEmpty(keys)) {
-        urlParams.forEach((value, key) => {
-            results[key] = value;
-        });
+        return lowerCaseUrlParams;
```

Comment:
This is going to have downstream errors. Technically speaking, `urlParams` isn't a dictionary that you can just swap out for `lowerCaseUrlParams`.

Reply:
before this line, we would assign results[key] = values from urlParams and return results later, I'm already doing that same logic in lines 288-291 expect I lower case them this time and return it when `keys` is empty, let me know if it still needs to be changed